src/main/java/net/raisetech/raisetechtask9/service/MovieServiceImpl.java

```diff
@@ -22,7 +22,7 @@ public List<Movie> findAll() {
 
     @Override
     public Optional<Movie> findById(int id) {
-        return movieMapper.findById(id);
+        return Optional.ofNullable(movieMapper.findById(id).orElseThrow(() -> new ResourceNotFoundException("resource not found")));
```

Comment:
業務処理の方に例外処理をさせる様に変更しました。

Reply:
movieMapper.findById(id)の戻り値はOptionalなのでOptional.ofNullableは不要ですね。
```suggestion
        return movieMapper.findById(id).orElseThrow(() -> new ResourceNotFoundException("resource not found"));
```